app/src/main/java/com/simply/birthdayapp/presentation/ui/components/LogoTopBar.kt

```diff
@@ -0,0 +1,39 @@
+package com.simply.birthdayapp.presentation.ui.components
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.simply.birthdayapp.R
+
+@Composable
+fun LogoTopBar() {
+    Row(
+        modifier = Modifier
+            .padding(vertical = 15.dp)
+            .fillMaxWidth(),
+        horizontalArrangement = Arrangement.Center,
+        verticalAlignment = Alignment.CenterVertically,
+    ) {
+        Image(
+            modifier = Modifier.size(width = 80.dp, height = 40.dp),
```

Comment:
We can specify it as `Modifier.width(80.dp).height(40.dp).`

Reply:
[Specified](https://github.com/SimplyTechnologies/android-internship/pull/14/commits/922b790e46068c41f481688be7fcfd5ba2084cea)